cmd/wallet-web/src/pages/WACIShare.vue

```diff
@@ -243,7 +241,8 @@ export default {
       if (!e.message.includes('12009')) {
         this.errors.push('Error initiating credential share');
       }
-      console.error('initiating credential share failed,', e);
```

Comment:
I think we should keep this for debugging purposes since we don't log this error anywhere else